src/bp-integrations/learndash/bp-learndash-filters.php

```diff
@@ -25,6 +25,20 @@
 
 add_action( 'admin_bar_menu', 'bb_group_wp_admin_bar_updates_menu', 99 );
 
+//user is removed from ld group
```

Comment:
@kent-carlo We should not inline function. It'll restrict to overwrite or remove an action if needed in the future.